app/jobs/heartbeat_job.rb

```diff
@@ -1,12 +1,13 @@
-# Writes "Heartbeat job performed" to the logs every minute. This message is
-# evidence that the worker is working properly. We can use a log alerting tool
-# to alert us if too much time passes without seeing one of these messages.
+require "net/http"
+
 class HeartbeatJob < CronJob
   self.cron_expression = "* * * * *"
 
   queue_as :heartbeat
 
   def perform
-    logger.info "Heartbeat job performed"
+    url = URI.parse(ENV["HEARTBEAT_CHECK_URL"])
```

Comment:
This may break in review as there won’t be monitoring and the env var will be absent. Can you plan for that? like if `ENV.key? "HEARTBEAT_CHECK_URL" {…}`